ui/components/app/transaction-list-item/transaction-list-item.component.js

```diff
@@ -176,7 +224,27 @@ function TransactionListItemInner({
         className={className}
         title={title}
         icon={
-          <TransactionIcon category={category} status={displayedStatusKey} />
+          ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+          isCustodian ? (
+            <div style={{ position: 'relative' }} data-testid="custody-icon">
+              <TransactionIcon
+                category={category}
+                status={displayedStatusKey}
+              />
+              <Icon
+                name={IconName.Custody}
+                className="transaction-list-item__icon-badge"
+                color={getTransactionColor(status)}
+                width={12}
+                height={12}
```

Comment:
I don't think we have these attributes exposed via API. but you can try  `size={IconSize.Xs}` to specify the size

Reply:
Fixed